test/measurability.lean

```diff
@@ -0,0 +1,76 @@
+/-
```

Comment:
Could you add in this file a test for `measurability?`

Reply:
I am not sure about how to test that form yet, but I will look at other test files and find something.

Reply:
I really don't know how I should test that. `measurability?` just does `show_term { measurability }` and prints the term generated by the tactic. Hence if `show_term` works as intended and `measurability` succeeds on a test, then `measurability?` shows a term that succeeds.
I cannot find tests for `show_term` nor tests for the specific form `tidy?`. The tests for `library_search` simply report the expected message in a comment but don't test that what is returned actually match it. And actually the test in test/library_search/ordered_ring.lean does not match.